pkg/util/helper/workstatus.go

```diff
@@ -246,18 +240,37 @@ func GetManifestIndex(manifests []workv1alpha1.Manifest, clusterObj *unstructure
 	return -1, fmt.Errorf("no such manifest exist")
 }
 
-func equalIdentifier(targetIdentifier *workv1alpha1.ResourceIdentifier, ordinal int, workload *unstructured.Unstructured) (bool, error) {
-	groupVersion, err := schema.ParseGroupVersion(workload.GetAPIVersion())
+// getManifestIndex gets the index of clusterObj in manifest list, if not exist return -1.
+func getManifestIndex(manifests []workv1alpha1.Manifest, objRef workv1alpha2.ObjectReference) (int, error) {
```

Comment:
I don't recommend adding such a tool function, it seems very redundant.

Reply:
but `GetManifestIndex` is also used by `work_status_controller` and has different `function parameter`

Reply:
maybe such arg `objRef workv1alpha2.ObjectReference` is not wanted.

Reply:
hi, can you check again? I I choose to reuse a new parameter.